src/extension.ts

```diff
@@ -231,9 +231,7 @@ async function activate(context: vscode.ExtensionContext) {
   context.subscriptions.push(
     vscode.commands.registerCommand('calva.loadFile', async () => {
       await eval.loadFile({}, config.getConfig().prettyPrintingOptions);
-      return new Promise((finished) => {
-        outputWindow.appendPrompt(finished);
-      });
+      outputWindow.appendPrompt();
```

Comment:
Could this be the reason the integration tests are now flaky?

Reply:
I don't think this change itself really produced a behavioral change, but I could be wrong. If it did though, maybe we should update the test, as I the behavior seems to be what's desired.

Reply:
The Promise there is unnecessary, but I can add it back and try the tests again when I have time.

Reply:
I don't think we should keep a promise just to make the test happy 😄 
But I think the promise might be useful outside the test as well. If you don't return that promise here, there is no way to wait for the file load + output window update to be done.

Reply:
I agree about the value of being able to wait for the window updates to be done.

Disagree about not keeping the promise for testing purposes only. 😄

Reply:
How about:

```typescript
      // We use this promise in the E2E tests
      return new Promise((finished) => {
         outputWindow.appendPrompt(finished);
       });
```

Reply:
I meant that I usually dislike having code that only exists for testing purposes. This promise is currently only used in tests, but it does make general sense for it to exist anyway. If I had a feature that wants to load a file and wait for that to be done, that promise is actually important. 
So I'm in favor of keeping it without the extra comment 😄

Reply:
And I meant that I usually like having code that exists only for testing. 😄 Testability is one of those altars I am ready to bring sacrifice to.

Reply:
> I don't think we should keep a promise just to make the test happy

I wasn't suggesting that - I just wanted to test and see if it was affecting the test for some reason.

The function inherently returns a promise since it's async, and if you await it, the promise is resolved after the `await loadFile` execution completes. There's no need to return a separate promise like that.

Reply:
Wow, somehow I missed that the promise was actually being resolved after the prompt was appended in the output window 😑 . I added it back as I agree it could be useful besides just for testing.